### DISCUSSION TITLE
Key management ideas

### DISCUSSION THREAD

**Discussion by tajhlande** [MrThearMan/sqlite3-cache#20] [Ideas]
Wondering whether you have any interest in more detailed key management features. something like:

- List all keys
- Regex search matching keys
- Clear keys matching regex

Maybe if you don't want to add the `sqlite3-pcre` library as a dependency, there could be simple prefix matching. I'm thinking about using and managing namespaced keys with dots between name path segments.

**Comment by MrThearMan** [comment]
I'd gladly welcome these features, although I would at first implement matching via the LIKE expression since it should cover your needs and doesn't require a dependency, as you said. You're welcome to submit a pull request! Otherwise, I might get to it at some point

**Reply by tajhlande** [reply]
In that case, I think I will give it a try.  I will be working on this in my spare time, so it might take a little while.  Do you want me to file this as a feature request first?
